inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/InlongConsumeServiceImpl.java

```diff
@@ -82,6 +84,36 @@ public Integer save(InlongConsumeRequest request, String operator) {
         return request.getId();
     }
 
+    @Override
+    public Integer saveBySystem(InlongGroupInfo groupInfo, String topic, String consumerGroup) {
+        String groupId = groupInfo.getInlongGroupId();
+        InlongConsumeEntity exists = consumeMapper.selectExists(groupId, topic, consumerGroup);
+        if (exists != null) {
```

Comment:
Maybe it is better to return the exists id?

Reply:
Good idea. Fixed.